go-client/api/swagger.yaml

```
@@ -0,0 +1,90 @@
+openapi: 3.0.2
+info:
+  title: Web Server
+  description: A simple web server for managing users
+  contact:
+    email: dmanor@redhat.com
+  version: "1.0"
+servers:
+- url: http://127.0.0.1:3000
+paths:
+  /api:
```

It's just an exercise of course, but be aware of versioning being an important part of every API
```suggestion
  /api/v1:
```